ex1/exercise_1_x_mix_drix.py

```diff
@@ -9,47 +9,57 @@ class PlayerToken(Enum):
     EMPTY = ' '
 
 
+X = PlayerToken.X
+O = PlayerToken.O
+
+
 class XOBoard:
     EMPTY = PlayerToken.EMPTY
 
     def __init__(self, board: List[List[PlayerToken]]):
         self._board = copy.deepcopy(board)
 
     def __str__(self) -> str:
-        """
-        Prints the board in a human-readable way.
-        """
-        pass
+        row1 = "|".join([i.value for i in self._board[0]])
+        row2 = "|".join([i.value for i in self._board[1]])
+        row3 = "|".join([i.value for i in self._board[2]])
+        return "\n".join([row1, row2, row3])
```

Comment:
Avoid code duplication !
How can we avoid code duplication here ?
* hint below







HINT: use `join` again